database/index.js

```diff
@@ -0,0 +1,33 @@
+const mongoose = require('mongoose');
+
+mongoose.connect('mongodb://localhost/menuService', { useNewUrlParser: true });
+
+const db = mongoose.connection;
+
+db.once('open', () => {
+  console.log('db connection for menu API open on port');
+});
+
+db.on('error', (err) => {
+  console.log('menu API db connection error: ', err);
+});
+
+const menuSchema = new mongoose.Schema({
+  restaurant_id: Number,
+  restaurant_name: String,
+  menus: [{
+    menuName: String,
+    sections: [{
+      sectionName: String,
+      menuItems: [{
+        menuItemName: String,
+        description: String,
+        prices: [],
```

Comment:
Double check here on this element of your schema. You are saying each menu item can have more than one price, right? Should this be: 
```suggestion
        prices: [Number],
```
?

Reply:
Or maybe
```suggestion
        prices: [String],
```